src/test/java/network/brightspots/rcv/TabulatorTests.java

```diff
@@ -194,6 +194,12 @@ static void setup() {
     Logger.setup();
   }
 
+  @Test
+  @DisplayName("aliases")
+  void aliases() {
+    runTabulationTest("aliases");
```

Comment:
Amazing test! But dear god the CDF format is hard to parse... is there a reason you chose to have the tests use that, rather than a more standard format?

But now that you've already done the work of creating it, maybe it's worth having both types of tests with the aliases (e.g. CDF and some xlsx or CSV input format)? This could be overkill, it's just really hard for me to vet that this test is actually WAI.

Reply:
I just grabbed a random test and copied/pasted. Happy to update with two additional formats testing.

Reply:
The only CSV input format I see is ClearBallot, where this doesn't apply: tabulation occurs by putting values in a `1` in a column corresponding to a unique candidate, so it seems that aliases are already consolidated. I could create a separate column with an alias, but that doesn't feel super useful.

I have gone ahead and added an XLSX test